content/6_Advanced/SCC.mdx

```diff
@@ -41,60 +52,61 @@ in-house explanation?
 
 ```cpp
 #include <bits/stdc++.h>
-using namespace std;
-
-using vi = vector<int>;
-#define pb push_back
 
-const int mx = 1e5 + 1;
+const int N = 1e5 + 1;
 
 // adj_t is the transpose of adj
-vi adj[mx], adj_t[mx], S;
-bool vis[mx];
-int id[mx];
+std::vector<int> adj[N], adj_t[N];
+std::vector<int> order;
+std::vector<int> vis(N), id(N);
+
+void dfs1(int v) {
+	vis[v] = true;
+	for (int u : adj[v]) {
+		if (!vis[u]) { dfs1(u); }
+	}
+	order.push_back(v);
+}
 
-void dfs(int x, int pass, int num = 0) {
+void dfs2(int x, int comp) {
```

Comment:
could you add a couple comments describing dfs1/2

Reply:
done.